mass_mailing_partner/models/mail_mass_mailing_contact.py

```diff
@@ -40,30 +40,38 @@ def _onchange_partner_mass_mailing_partner(self):
 
     @api.model
     def create(self, vals):
-        record = self.new(vals)
+        # Ensure that defaults are loaded (e.g.: import csv or xls)
+        values_w_defaults = self.default_get(self._fields.keys())
+        values_w_defaults.update(vals)
+        record = self.new(values_w_defaults)
         if not record.partner_id:
             record._set_partner()
         record._onchange_partner_mass_mailing_partner()
         new_vals = record._convert_to_write(record._cache)
         new_vals.update(
-            subscription_list_ids=vals.get('subscription_list_ids', False),
-            list_ids=vals.get('list_ids', False)
+            subscription_list_ids=values_w_defaults.get(
+                'subscription_list_ids', False),
+            list_ids=values_w_defaults.get('list_ids', False)
         )
-        return super(MailMassMailingContact, self).create(new_vals)
+        return super().create(new_vals)
 
     def write(self, vals):
+        # Ensure that defaults are loaded (e.g.: import csv or xls)
+        values_w_defaults = self.default_get(self._fields.keys())
```

Comment:
I'm pretty sure this is a bad idea.

You'd be resetting every field to their defaults if it's not specified on a call to `write()`, and that's pretty common. :thinking:

Reply:
Yes, indeed this is something I haven't thought about. Any way, why writing you need to check that context?

Reply:
Well, I'm guessing that with an import you could update existing records as well. But to be honest I didn't test if there was an issue in that case

Reply:
But doing an import you don't need to change the list. You are modifying other values (like email or similar). It's better to remove this part in the update.